addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -266,15 +306,9 @@ export class Many2ManyTagsFieldColorEditable extends Many2ManyTagsField {
         canEditColor: true,
     };
 
-    getTagProps(record) {
-        const props = super.getTagProps(record);
-        props.onClick = (ev) => this.onBadgeClick(ev, record);
-        return props;
-    }
-
     onBadgeClick(ev, record) {
         if (!this.props.canEditColor) {
-            return;
```

Comment:
this feature completely conflicts with the color selection...

Reply:
I did made sure when canEditColor is true then do not open form view dialog
So it does not break existing color flow